lib/modulesync/util.rb

```diff
@@ -12,7 +12,7 @@ def self.symbolize_keys(hash)
 
     def self.parse_config(config_file)
       if File.exist?(config_file)
-        YAML.load_file(config_file) || {}
+        YAML.load_file(config_file, aliases: true) || {}
```

Comment:
Older versions of ruby do not have support for this keyword.  Given we will drop Puppet 7 ([discussion](https://github.com/voxpupuli/community-triage/issues/60)), testing with these old ruby versions do not add value and I would rather remove them.